lilcom/lilcom_c_extension.c

```diff
@@ -190,6 +189,11 @@ int decompress_int16_internal(int num_axes, int axis,
                               int16_t *output_data,
                               PyObject *input, PyObject *output) {
   assert(axis >= 0 && axis < num_axes);
+
+
+  /* ISSUE: the function is supposed to be returning a conversion exponent and an error code, to do this we must make a consideration to handel it here and in the python wrapper  */
+  int conversion_exponent = 0;
+
   int dim = PyArray_DIM(input, axis),
```

Comment:
We should debate here in the near future on how to handle this return value. but for now, it's just one simple resolvable issue.